File: EnglishTraining/models/VmWord.cs
using System.Collections.Generic;

namespace EnglishTraining
{
    public class VmWord
    {
        public string Name_en { get; set; }
        public string Name_ru { get; set; }
        public bool FourDausLearnPhase { get; set; }
        public int LearnDay { get; set; }
        public int RepeatIterationNum { get; set; }
        public string NextRepeatDate { get; set; }
        public int DailyReapeatCountForEng { get; set; }
        public int DailyReapeatCountForRus { get; set; }
        public IList<VmDictor> Dictors_en { get; set; }
        public IList<VmDictor> Dictors_ru { get; set; }
    }
}

File: EnglishTraining/Frontend/track-list/components/models/VmWord.ts
import { VmDictor } from "./VmDictor";
import { VmName } from "./VmName";

export interface VmWord {
    Name: VmName;
    FourDausLearnPhase: boolean;
    LearnDay: number;
    RepeatIterationNum: number;
    NextRepeatDate: string;
    DailyReapeatCountForEng: number;
    DailyReapeatCountForRus: number;
    Dictors_en?: VmDictor[];
    Dictors_ru?: any;
    // Dictors_ru?: VmDictor[];
}

